release_bot/releasebot.py

```diff
@@ -217,10 +220,15 @@ def make_new_pypi_release(self):
 
         def release_handler(success):
             result = "released" if success else "failed to release"
-            msg = f"I just {result} version {self.new_release['version']} on PyPI"
+            msg = ""
```

Comment:
You can cut this line, msg will be set no matter what

Reply:
Roger.